jetstream/ordered.go

```diff
@@ -142,6 +145,9 @@ func (c *orderedConsumer) Consume(handler MessageHandler, opts ...PullConsumeOpt
 			select {
 			case <-c.doReset:
 				if err := c.reset(); err != nil {
+					if errors.Is(err, errOrderedConsumerClosed) {
```

Comment:
The only case for `errOrderedConsumerClosed` is when its stopped, so there is no point to try resetting it again, right?

Reply:
Yes, there is no point. I used `continue` here because when the consumer is stopped we'll eventually end up in `<-sub.done` case where the actual cleanup is performed.